SU-SWS/decanter
# Add 4 new @modular-spacing steps (#6 to #9) and modify 2 existing steps (#3 and #4)

# READY FOR REVIEW

# Summary
- Size of existing @modular-spacing between step 3 and 4, 4 and 5 are too big, need to add in-between spacers, and add additional spacers to account for wider range of spacing needs.
![Screen Shot 2019-10-01 at 2 15 50 PM](https://user-images.githubusercontent.com/42749717/66001634-b9616900-e456-11e9-80a1-51b37815695f.png)

## New spacers:

@modular-spacing 4 (change)
3XL: Base * 3.2

@modular-spacing 5 (change)
4XL: Base * 4

@modular-spacing 6 (new)
5XL: Base * 5

@modular-spacing 7 (new)
6XL: Base * 6

@modular-spacing 8 (new)
7XL: Base * 7

@modular-spacing 9 (new)
8XL: Base * 9

- Round the result modular-spacing values to 2 decimals.

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch and compile styleguide
2. Test the `@modular-spacing` mixin by passing in the newly added integer steps, e.g., in `scss/components/_cards.scss`, apply `@modular-spacing('padding', 9 8 7 6)` to `.su-card__contents`, check the rendered padding value matches the ones in the screenshot above at all 3 breakpoint ranges.

# Affected Projects or Products
- Decanter
- Redwood

# Associated Issues and/or People
- #492 
- @kerri-augenstein
